appyx-interactions/common/src/commonMain/kotlin/com/bumble/appyx/transitionmodel/BaseInterpolator.kt

```diff
@@ -96,6 +95,40 @@ abstract class BaseInterpolator<NavTarget : Any, ModelState, Props>(
         }
     }
 
+    private fun onAnimationStarted(
+        elementProps: Props,
+        targetProps: MatchedProps<NavTarget, Props>
+    ) {
+        val isStarted = animations.getOrDefault(targetProps.element.id, false)
+        if (elementProps.isEqualTo(targetProps.props).not() && !isStarted) {
```

Comment:
```suggestion
        if (elementProps.isNotEqualTo(targetProps.props) && !isStarted) {
```

Could be a more understandable naming like this just with an extra method in the interface